api/conf.py

```diff
@@ -10,6 +10,7 @@ class BaseConfig(metaclass=MetaFlaskEnv):
     DEBUG = False
     TESTING = False
     SERVICE_NAME = 'shared-db-channel'
+    HUB_URL = 'http://127.0.0.1:5001/'
```

Comment:
better to be an env variable

Reply:
it is already (https://github.com/brettlangdon/flask-env)